bin/paragon-scripts.js

```diff
@@ -47,12 +49,21 @@ const COMMANDS = {
     executor: helpCommand,
     description: 'Displays help for available commands.',
   },
+  version: {
+    executor: versionCommand,
+    description: 'Displays the current version of Paragon CLI.',
+  },
 };
 
 (async () => {
   const [command] = process.argv.slice(2);
   const executor = COMMANDS[command];
 
+  if (VERSION_COMMANDS.includes(command)) {
+    versionCommand();
+    return;
+  }
+
```

Comment:
Let's not do this, this breaks the whole pattern we have for CLI commands - you should never call the command directly, but instead call `executor` method that you defined in `COMMANDS` object.

Instead, create an aliases map for command, e.g.
```js
// add this at the top of the file
commandAliases = {
  '-v': 'version',
  '--version': 'version',
}

// and extract executor this way
const [command] = process.argv.slice(2);
const resolvedCommand = commandAliases[command] || command;
const executor = COMMANDS[resolvedCommand];
```

then you don't need to add any `if` block and it will be very easy to add aliases for the future commands as well.

Reply:
Ok, thanks